common/setups/hybrid/util.py

```diff
@@ -376,34 +407,81 @@ def __init__(
             'alignment': "train_vtln",  # if using run function not needed
         }
         ##################################################
-        :param vtln_sat_recognition_args:
+        :param recognition_args:
         ##################################################
         """
-        self.vtln_sat_training_args = vtln_sat_training_args
-        self.vtln_sat_recognition_args = vtln_sat_recognition_args
-        self.sdm_vtln_sat_args = sdm_vtln_sat_args
+        self.training_args = training_args
+        self.recognition_args = recognition_args
+        self.sdm_args = sdm_args
+
+
+class ForcedAlignmentArgs:
+    def __init__(self, name, target_corpus, flow, feature_scorer):
```

Comment:
No, because atm in the `run` function the feature scorer would be taken from all `train_corpora` if feature_scorer is not a FeatureScorer Object.
Idea would be to apply all feature scorers from all train_corpora to the new corpus.

Reply:
I was asking because in your suggested change `rasr_system.py:171` in function
```
   def forced_align(self, name, corpus, target_corpus, flow, feature_scorer, **kwargs):
```

there is the additional `corpus` argument. So I was wondering if by symmetry we need this also here, or could omit it there.

Reply:
Ah OK. the function parameter `corpus` is needed but specified in the run function.